src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.ts

```diff
@@ -155,10 +152,14 @@ export class CephfsSubvolumegroupFormComponent extends CdForm implements OnInit
           task: new FinishedTask('cephfs/subvolume/group/' + URLVerbs.EDIT, {
             subvolumegroupName: subvolumegroupName
           }),
-          call: this.cephfsSubvolumeGroupService.update(
```

Comment:
are you sure, you want to change from update -> create ?

Reply:
the update is mainly used for resizing of the subvolume group, and if the same is achieved by the create, then I guess its okay.